Makefile

```diff
@@ -44,17 +44,25 @@ PLATFORMS := darwin linux windows
 
 GO_LDFLAGS_WIN = -ldflags "-X github.com/mendersoftware/mender-artifact/cli.Version=$(VERSION) -linkmode=internal -s -w -extldflags '-static' -extld=x86_64-w64-mingw32-gcc"
 
-build-natives:
+build-native-linux:
+	 @arch="amd64";
+	 @echo "building linux";
+	 @env GOOS=linux GOARCH=$$arch \
+        $(GO) build -a $(GO_LDFLAGS) $(BUILDV) $(BUILDTAGS) -o $(PKGNAME)-linux ;
```

Comment:
There is a mix of spaces and tabs (it works fine because the line with spaces is preceded by `\`)-